### ISSUE TITLE
Build from source internal flash.

### ISSUE THREAD

**Issue by mattridrm** [pazi88/STM32_mega#5]
Does using the recommended VScode build instructions build for the STM32 internal flash?  Can the internal flash be extended larger than the default speeduino  4095 bytes? I am modifying the firmware and would like to store some extra tables.  Thanks for your work on this adaptor board.

**Comment by pazi88** [comment]
Yes, by default Speeduino code is built by using internal flash. It should be possible to increase size of the tune-memory, but looking at the code, I'm not sure how to do that.
